pysingfel/solvent_form_factor/tests/libsaxs_take2_test.py

```diff
@@ -42,9 +42,9 @@ def setUp(self):
         #sys.exit()
         #def test_assign_form_factors_2_profiles(self):
 
-        self.prof, water_ff,r_size = ps.solvent_form_factor.saxs_profile.assign_form_factors_2_profile(self.particles,self.prof,self.saxs_sa,self.vff,self.dff,self.ft,self.num_atoms,self.r_size,self.verbose)
+        self.prof, water_ff,r_size = ps.solvent_form_factor.saxs_profile.assign_form_factors_2_profile(self.particles,self.prof,self.saxs_sa,self.vff,self.dff,self.ft,self.num_atoms,self.r_size)
 
-        self.intensity = ps.solvent_form_factor.saxs_profile.calculate_profile_partial(self.prof, self.particles, self.saxs_sa, self.ft, self.vff,self.dff,self.verbose, c1,c2)
+        self.intensity = ps.solvent_form_factor.saxs_profile.calculate_profile_partial(self.prof, self.particles, self.saxs_sa, self.ft, self.vff,self.dff, c1,c2)
 
     
     def test_vacuum_form_factor_data(self):
```

Comment:
unit tests should print to screen when the test fails, otherwise these print statements should be comments

Reply:
OK, done.